tvb_build/docker/Dockerfile-run

```diff
@@ -51,7 +52,8 @@ RUN cd tvb-root; \
     /bin/bash -c "source activate tvb-run && /bin/bash install_full_tvb.sh"
 
 COPY --chown=$TVB_USER:$TVB_USER .tvb.configuration $USER_HOME/.tvb.configuration
-COPY --chown=$TVB_USER:$TVB_USER step1/_help $USER_HOME/tvb-root/framework_tvb/tvb/interfaces/web/static/help
+COPY --chown=$TVB_USER:$TVB_USER .tvb.configuration /root/.tvb.configuration
+#COPY --chown=$TVB_USER:$TVB_USER step1/_help $USER_HOME/tvb-root/framework_tvb/tvb/interfaces/web/static/help
```

Comment:
This copy line is usefull

Reply:
It is, it just doesn't work on my local builds.

Reply:
I did downloaded step1 ZIP from Jenkins on my machine for this to work. 
It should be ignored already by Git, so it will not hurt if you have it locally.